dlk/tests/test_code_generation_x86_64.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Test file for code generation"""
+from nose2.tools import params
+import os
+from test_code_generation_base import TestCodeGenerationBase, get_configurations_by_architecture
+from test_code_generation_base import get_configurations_by_test_cases, dict_codegen_classification
+from tstconf import CURRENT_TEST_LEVEL
+from tstutils import updated_dict, TEST_LEVEL_FUTURE_TARGET
+import unittest
+
+
+def get_configurations():
+    cpu_name = "x86_64"
+    test_cases = [
+        {'use_avx': True, 'hard_quantize': True, 'threshold_skipping': True},
+        {'use_avx': True, 'hard_quantize': True, 'threshold_skipping': False},
+        {'use_avx': True, 'hard_quantize': False, 'threshold_skipping': False},
+        {'use_avx': False, 'hard_quantize': True, 'threshold_skipping': True},
+        {'use_avx': False, 'hard_quantize': True, 'threshold_skipping': False},
+        {'use_avx': False, 'hard_quantize': False, 'threshold_skipping': False},
```

Comment:
There are test cases for x86_64.

| use_avx | hart_quantize | threshold_skipping |
| --- | --- | --- |
| o | o | o |
| o | o | x |
| o | x | o |
| x | o | o |
| x | o | x |
| x | x | o |

There was no case of hard_quantize:`False` and threshold_skipping:`False`
I am not sure about these test cases, so I am following the previous test cases.
@iizukak 
If we need tests with the case hard_quantize:`False` and threshold_skipping:`False`, please let me know, I will add them.

Reply:
@hadusam Thanks. We don't need Both `False` case.

Reply:
@iizukak Thank you! Then, plz review this PR 😄